src/components/common/modal/Modal.jsx

```diff
@@ -9,14 +10,15 @@ import CloseRoundedIcon from "@mui/icons-material/CloseRounded";
 
 const Modal = (props) => {
   const [display, setDisplay] = useState(false);
+  const shouldMount = useDelayBoolean(props.display, theme.duration_num);
 
   useDisableScroll(props.display);
   useKeyboardOperation({
     display: props.display,
     onEnter: props?.onEnter,
     onEscape: props.onClickClose,
   });
-  useEffect(() => setDisplay(props.display), [props.display]);
+  useEffect(() => setDisplay(props.display), [shouldMount]);
```

Comment:
```suggestion
  useEffect(() => setDisplay(shouldMount), [shouldMount]);
```

useEffect 안에도 `shouldMount` 로 바뀌어야 하지 않을까요?